tests/upsert.test/t01_upsert.expected

```diff
@@ -143,3 +143,35 @@ done
 done
 done
 done
+(part='---------------------------------- PART #07 ----------------------------------')
+done
+done
+done
+done
+done
+done
+done
+done
+(rows inserted=1)
+done
+done
+(i=1)
+done
+(i=2)
+done
+(part='---------------------------------- PART #08 ----------------------------------')
+done
+done
+done
+done
+done
+done
+done
+done
+done
+done
+[commit] failed with rc 2 add key constraint duplicate key 'COMDB2_PK' on table 't1' index 0
```

Comment:
Could we return 299 (DUPLICATE), instead?

Reply:
See other comment. I could change the message to make it clear on the client end that it is an uncommittable transaction (and therefore failed verify) due to the duplicate key violation, or I could get rid of all of the uncommittable flags and just have it be a regular duplicate key violation error.

Reply:
This is fine!